macros/NEXT100_Ar_alpha_full.config.mac

```diff
@@ -0,0 +1,56 @@
+## ----------------------------------------------------------------------------
+## nexus | NEXT100_Ar_alpha_full.config.mac
+##
+## Initialization macro to alpha particle decays in the active
+## volume of the NEXT-100 detector during the low pressure run.
+##
+## The NEXT Collaboration
+## ----------------------------------------------------------------------------
+
+##### VERBOSITY #####
+/run/verbose 0
+/event/verbose 0
+/tracking/verbose 0
+
+/process/em/verbose 0
+
+##### GEOMETRY #####
+/Geometry/Next100/elfield true
+/Geometry/Next100/EL_field 6.7 kV/cm
+/Geometry/Next100/max_step_size 1. mm
+/Geometry/Next100/pressure 4.2 bar
+/Geometry/Next100/gas GAr
+/Geometry/PmtR11410/time_binning 25. nanosecond
+/Geometry/Next100/sipm_time_binning  1. microsecond
+
+## Geant4 only permits units of mm/ns via the macro
+/Geometry/Next100/drift_vel   1.480 mm/mus
+/Geometry/Next100/ELdrift_vel 6.249 mm/mus
+
+/Geometry/Next100/drift_transv_diff 1.87 mm/sqrt(cm)
+/Geometry/Next100/drift_long_diff 0.65 mm/sqrt(cm)
+
+/Geometry/Next100/ELtransv_diff 0.46 mm/sqrt(cm)
+/Geometry/Next100/ELlong_diff 0.30 mm/sqrt(cm)
+
+
+##### GENERATOR #####
+/Generator/SingleParticle/particle alpha
+/Generator/SingleParticle/min_energy 5.6 MeV
+/Generator/SingleParticle/max_energy 5.6 MeV
+/Generator/SingleParticle/region ACTIVE
+
+##### ACTIONS #####
+/Actions/DefaultEventAction/min_energy 50 keV
+/Actions/DefaultEventAction/max_energy 6 MeV
+
+##### PHYSICS #####
+## No full simulation
```

Comment:
This comment is misleading, since this is actually full simulation. Please, change it,